<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana 2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255,255, 255);</string>
  </property>
  <widget class="QPushButton" name="prev_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Atras</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>----------------------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>131</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/bahia_de_linea.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LL">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_PR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TT">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TO">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SL">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TC">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_IN">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_BB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BB</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
